pkg/apis/autoscaling/v1alpha1/pa_lifecycle.go

```diff
@@ -84,10 +84,10 @@ func (pa *PodAutoscaler) annotationFloat64(key string) (float64, bool) {
 // not set.
 // Note: min will be ignored if the PA is not reachable
 func (pa *PodAutoscaler) ScaleBounds(asConfig *autoscalerconfig.Config) (int32, int32) {
-	var min, max int32
 
+	var min int32
 	if pa.Spec.Reachability != ReachabilityUnreachable {
-		min, _ = pa.annotationInt32(autoscaling.MinScaleAnnotationKey)
+		min, _ := pa.annotationInt32(autoscaling.MinScaleAnnotationKey)
```

Comment:
this is now a no-op.. I'm kind of surprised if the tests didn't fail with this though, maybe we need to add a test?

Reply:
LGTM put out a warning; and the test did fail for me locally when I tried now. kinda of losing track of all the tests on this PR with all the recent changes/pushes

Reply:
in general quite good to run all with `go test ./...` as a final test before pushing

Reply:
sounds good, thanks for the thorough review and feedback; i'm new to golang so it really helps.

Reply:
yvw, I remember being new to Golang _all too well_ ;)